analyzer/analyzer.spicy

```diff
@@ -9,7 +9,7 @@ public type File = unit {
 };
 
 type Chunk = unit(inout top: PNG::File) {
-    length: uint32;
+    length: uint32 &requires=($$ <= 2**31);
```

Comment:
Hmm ... not sure this helps much with anything? Why not directly address #6 by picking some  '"reasonable"
value'?

Reply:
I added a patch making this customizable (with a Zeek limit of 128 MiB), but still kept the default size limit of `<= 2**31` since it comes from the protocol spec. With https://github.com/zeek/spicy/pull/2036 we should not see immediate OOMs anymore.